tests/test_internals.py

```diff
@@ -34,3 +34,14 @@ def test_add_landmark_with_title():
 	add_landmark(dom, "file", landmarks)
 
 	assert landmarks[0].title == "Bar"
+
+def test_inner_text():
```

Comment:
Great, glad to see we're getting some tests in here. Do you have to also rebuild the test golden files when adding a test?

Reply:
Not sure: I added this test and re-ran `pytest` and everything worked fine.